lib/crystalball/predictor/associated_specs.rb

```diff
@@ -6,15 +6,16 @@ class Predictor
     # When used will look for files matched to `from` regex and use captures to fill `to` string to
     # get paths of proper specs
     class AssociatedSpecs
+      ALL = './'
+
```

Comment:
not used

Reply:
I'm going to use it here https://github.com/toptal/crystalball-toolbox/pull/3/files#diff-b8ee0ab8664c89405a7e92af9c552629R17

Reply:
I believe it's project specific. some projects require `./` others `./spec` others `./test` etc. So it shouldn't be here